README.md

```diff
@@ -4,15 +4,51 @@ Spokestack provides an extensible speech recognition pipeline for the iOS
 platform. It includes a variety of built-in speech processors for Voice
 Activity Detection (VAD), wakeword activation, and Automatic Speech Recognition (ASR).
 
+## Features
+
+  - Voice activity detection
+  - Wakeword activation with three different models
+  - Simplified Automated Speech Recognition interface
+  - Speech pipeline seamlessly integrates VAD-trigged wakeword detection using on-device machine learning models into ASR utterances.
+
 ## Installation
 [![](https://img.shields.io/cocoapods/v/Spokestack-iOS.svg)](https://cocoapods.org/pods/Spokestack-iOS)
 
+[CocoaPods](https://cocoapods.org) is a dependency manager for Cocoa projects. For usage and installation instructions, visit their website. To integrate Spokestack into your Xcode project using CocoaPods, specify it in your Podfile:
+
 `pod 'Spokestack-iOS'`
 
-[CocoaPods](https://cocoapods.org) is a dependency manager for Cocoa projects. For usage and installation instructions, visit their website. To integrate Spokestack into your Xcode project using CocoaPods, specify it in your Podfile:
+## Usage
+
+### Configure Wakeword-activated Automated Speech Recognition
+
+ ```
+ // assume that self implements the SpeechEventListener and PipelineDelegate protocols
+ let pipeline = SpeechPipeline(SpeechProcessors.appleSpeech.processor,
+                                speechConfiguration: SpeechConfiguration(),
+                                speechDelegate: self,
+                                wakewordService: SpeechProcessors.appleWakeword.processor,
+                                pipelineDelegate: self)
+ pipeline.start()
+ ```
+
+This example creates a speech recognition pipeline using an ASR-based wakeword detector that is triggered by VAD, which in turn activates Apple Speech based ASR, returning the resulting utterance to the `SpeechEventListener` event listener.
+
+### Reference implementation
+
+The `SpokestackFrameworkExample` project contains reference implementations for how to use the Spokestack library, along with runnable examples of the wakeword and ASR components.
+
+See `SpeechPipeline` and `SpeechConfiguration` for further configuration documentation
+
+## Development
+
+### Preconditions
+
+  1. Ensure that CocoaPods has been installed (via `gem`, not via `brew`).
+  2. Ensure that jazzy has been installed via `gem`.
```

Comment:
Commands for each of these installations? You could list both of them using `gem install` and note that homebrew is not recommended either at the beginning or end of the section (maybe saying why it's the wrong choice).